pom.xml

```diff
@@ -55,7 +55,7 @@
 		<!-- for backwards compatibility, use maven-checkstyle-plugin.version -->
 		<checkstyle.version>${maven-checkstyle-plugin.version}</checkstyle.version>
 		<puppycrawl-tools-checkstyle.version>9.2.1</puppycrawl-tools-checkstyle.version>
-		<spring-javaformat.version>0.0.29</spring-javaformat.version>
+		<spring-javaformat.version>0.0.34</spring-javaformat.version>
```

Comment:
in `spring-cloud-kubernetes` we added some jdk-17 feature and the `0.0.29` version of the formatter plugin formats that code in not a very good way. For example:

```
else if (source instanceof BootstrapPropertySource<?>bootstrapPropertySource) {
```

instead of:

```
else if (source instanceof BootstrapPropertySource<?> bootstrapPropertySource) {
```

Or:

```
NormalizedSource permits NamedSecretNormalizedSource,LabeledSecretNormalizedSource
```

instead of : 

```
NormalizedSource permits NamedSecretNormalizedSource, LabeledSecretNormalizedSource
```

With version `34` - we do get the nicer output. I am not sure of the implications in other projects, though, from this bump. May be someone more experienced can comment more.